Doc/conf.py

```diff
@@ -263,6 +263,24 @@
     r'https://github.com/python/cpython/tree/.*': 'https://github.com/python/cpython/blob/.*'
 }
 
+linkcheck_anchors_ignore = [
+    # ignore anchors that starts with a '/', e.g. Wikipedia media:
+    # https://en.wikipedia.org/wiki/Walrus#/media/File:Pacific_Walrus_-_Bull_(8247646168).jpg
+    r'\/.*',
+]
+
+linkcheck_ignore = [
+    # The crawler gets "Anchor not found"
+    r'https://github.com.+?#.*',
+    r'https://devguide.python.org.+?/#.*',
+    r'https://developer.apple.com/documentation/.+?#.*',
```

Comment:
Let's alphabetise this section, but makes it easier to diff other projects if they also have them:

```suggestion
    r'https://developer.apple.com/documentation/.+?#.*',
    r'https://devguide.python.org.+?/#.*',
    r'https://github.com.+?#.*',
```